tests/mocks/common-password.ts

```diff
@@ -0,0 +1,8 @@
+import { http, HttpResponse } from 'msw'
```

Comment:
Let's follow the convention of other services. Call the file: `pwnedpasswords.ts` and export an array of handlers called `handlers`.